src/toast/toast.test.tsx

```diff
@@ -124,6 +124,24 @@ describe('useToast', () => {
         expect(within(screen.getByRole('alert')).getByTestId('test-icon')).toBeInTheDocument()
     })
 
+    it('allows to keep the toast after the action is performed if closeToast is false', () => {
```

Comment:
I'd suggest putting this in a `describe` block to make it more readable:

```js
describe('if `closeToast` is false', () => {
    it('keeps the toast visible after the action button is clicked', () => {
        // …
    })
})
```

also, there should be a sister test to this one, testing that when `closeToast` is omitted or true, the toast is dismissed when clicking the action button.